### ISSUE TITLE
Emails not saving for Lead or Person

### ISSUE THREAD

**Issue by jferdizzle** [salespreso/prospyr#12]
I've just started investigating the issue, but my belief is that there seems to be an issue with the field `Email`. When I try instantiating an Email field, I get a recursive error. I believe if you edit the EmailSchema's Email field to be `email = fields.String(allow_none=True)`, this should work for Lead and Person.

To test what I am referring to, try following the docs example for creating a person and the person 'steve' will not have an email.

```
from prospyr import Person

steve = Person(
    name='Steve Cognito',
    emails=[{'category': 'work', 'email': 'steve@example.org'}]
)

# steve only exists locally at this stage
steve.id
>>> None

# now he exists remotely too
steve.create()
>>> True
steve.id
>>> 1
```

**Comment by bennullgraham** [comment]
You're right, this is broken. First guess is my hack to deal with ProsperWorks allowing whitespace around email addresses (`prospyr/fields.py:35`) is not working as expected (or no longer working as expected).
